lldb/test/Shell/Commands/TestImageLookupColor.test

```diff
@@ -0,0 +1,23 @@
+UNSUPPORTED: system-windows
+
+# RUN: %clang_host -g %S/Inputs/main.c -o %t
+# RUN: %lldb %t -b -o 'image lookup -r -s ma' | FileCheck %s
+
+# CHECK: 3 symbols match the regular expression 'ma' in {{.*}}
+# The [[ confuses FileCheck so regex match it.
+# CHECK-NEXT:         Name: {{.+}}31mma{{.+}}0min.c
```

Comment:
Only the lines that are supposed to have colour are important here, and ordering isn't really an issue.

Keep the "3 symbols match" check, just as a sanity check in case the output changes, but then only check for the lines with colour (you won't need -NEXT).

Reply:
Well, ordering is something we want to be correct - but that's tested by other tests. You can assume it will be correct.